Userland/Applications/Browser/CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ serenity_component(
     DEPENDS BrowserSettings ImageDecoder RequestServer WebContent WebSocket
 )
 
+compile_ipc(WebDriverSessionServer.ipc WebDriverSessionServerEndpoint.h)
+compile_ipc(WebDriverSessionClient.ipc WebDriverSessionClientEndpoint.h)
```

Comment:
You'll need to duplicate this in Lagom CMake lists. That or put the IPC files in the web driver folder and add a manual dep to Lagom CMake lists